recipes/crms2map/meta.yaml

```diff
@@ -0,0 +1,108 @@
+{% set name = "crms2map" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name |lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jinikeda/{{ name |lower }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 7c718dbe50dd3fe7cb2cf294418e846028126ed6a4f9c3960db92b7cfce3ae53
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  # More info about selectors can be found in the conda-build docs: 
+  # https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors
+  noarch: generic
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    #- {{ compiler('c') }}
+  host:
+    - python >=3.9
+    - pip
+    - setuptools
+  run:
+    - python >=3.9
+    - numpy
+    - pandas
+    - geopandas
+    - rasterio
+    - scipy
+    - xarray
+    - openpyxl
+    - rioxarray
+    - folium
+    - seaborn
+    - matplotlib-base
+    - pyproj
+    - scikit-learn
+    - click
+    - fiona
+    - shapely
+    - lz4-c
+    - zlib
+    - openssl
+    - libtiff
+    - libgdal
+    - sqlite
+    - pytest
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - src.CRMS_general_functions
+    - src.CRMS_Discrete_Hydrographic2subsets
+    - src.CRMS2Resample
+    - src.CRMS2Plot
+    - src.click_main
+  # For python packages, it is useful to run pip check. However, sometimes the
+  # metadata used by pip is out of date. Thus this section is optional if it is
+  # failing.
+  requires:
+    - pytest
+    - pytest-mock
+    - click
```

Comment:
You don't need to list requirements here that are already in the main `run` requirements. Those get pulled in automatically.

Reply:
I adjusted the test requirements.